test/tests_for_auth/ee2_authutil_test.py

```diff
@@ -6,11 +6,14 @@
 from lib.execution_engine2.authorization.roles import AdminAuthUtil
 from lib.execution_engine2.exceptions import AuthError
 
+from test.utils_shared.test_utils import bootstrap
```

Comment:
I dont think this line is necessary any more?

Reply:
fixed here https://github.com/kbase/execution_engine2/pull/289/files#r569835471